### ISSUE TITLE
Optimization spend less battery

### ISSUE THREAD

**Issue by EnzoDantas** [AOF-Dev/MCinaBox#798]
Optimization spend less battery Focus more on those things

**Smartphone (please complete the following information):**
 - Device: [e.g. Samsung Galaxy A7 2018]
 - OS: [e.g. Android 8.1]
 - App Version [e.g. v0.1.4p5]

**Additional context**

**Comment by Adam0530** [comment]
Yeah It does use much b3... but I have never have an issue like overheat when playing minecraft using mcinabox... Pretty good for me
